docs/product-user-guide/src/main/asciidoc/interacting-with-processes-starting-stopping-tasks-proc.adoc

```diff
@@ -13,15 +13,20 @@ Ensure that you are logged in and have appropriate permissions for starting and
 
 .Procedure
 . Go to *Menu* -> *Track* -> *Task Inbox*.
-. On the *Task Inbox* page, click on the task to open it.
+. On the *Task Inbox* page, click the task to open it.
 . On the *Work* tab of the task page, click *Start*. Once you start a task, its status changes to *InProgress*.
++
+You can view the status of tasks on the *Task Inbox* as well as on the *Manage Tasks* page.
 
- You can view the status of tasks on the *Task Inbox* as well as on the *Manage Tasks* page. Note that only users with admin and process admin role can access the *Manage Tasks* page.
+[NOTE]
+====
+Only users with admin or process-admin role can access the *Manage Tasks* page.
```

Comment:
Although correct, this statement is incomplete and can cause confusion. 
Only users with _process-admin_ role can actually see the task list with out-of-the-box setup. Users with _admin_ role can access the page as well, but will always see an empty task list, if an additional role permission setup is not performed on Process Server side.

Reply:
Change Ack.